tests/std/tests/P0896R4_ranges_alg_copy_n/test.cpp

```diff
@@ -9,34 +9,27 @@
 
 #include <range_algorithm_support.hpp>
 
-constexpr void smoke_test() {
-    using ranges::copy_n, ranges::copy_n_result, ranges::iterator_t;
-    using std::same_as;
+using namespace std;
+using same_as;

```

Comment:
Wat?
```suggestion
```

Reply:
Narf, it was late and I was busy trying to take over the world